KarhooUISDK/Controllers/Payment/AddCard/AdyenCardRegistrationFlow.swift

```diff
@@ -97,6 +97,7 @@ final class AdyenCardRegistrationFlow: CardRegistrationFlow {
         let configuration = DropInComponent.Configuration(apiContext: apiContext)
         configuration.card.showsStorePaymentMethodField = showStorePaymentMethod
         configuration.card.showsHolderNameField = true
+        configuration.localizationParameters = LocalizationParameters(bundle: Bundle(for: Self.self))
```

Comment:
If it will work if you use Bundle.current instead of Bundle(for: Self.self)??

Reply:
Changed to .current